bentoml/yatai/yatai_service_impl.py

```diff
@@ -300,7 +300,10 @@ def GetBento(self, request, context=None):
             bento_metadata_pb = self.bento_metadata_store.get(
                 request.bento_name, request.bento_version
             )
+
             if bento_metadata_pb:
+                uri = self.repo.get(bento_metadata_pb.name, bento_metadata_pb.version)
```

Comment:
add a comment here explaining that s3 link might be expired? which I assume is why we needed this line here